src/test/java/io/confluent/maven/resolver/KafkaFetchTest.java

```diff
@@ -0,0 +1,32 @@
+package io.confluent.maven.resolver;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import org.eclipse.aether.resolution.VersionRangeRequest;
+import org.eclipse.aether.resolution.VersionRangeResult;
+import org.eclipse.aether.version.Version;
+import org.eclipse.aether.util.version.GenericVersionScheme;
+
+public class KafkaFetchTest {
+    private static final String CCS = "ccs-kafka";
+    private static final String constrainText = "[6.0.1-1, 6.0.2-1)";
```

Comment:
why does the version range used for this test not include the -ce or -ccs qualifiers?